lib/models/category.rb

```diff
@@ -3,5 +3,5 @@ class Category
 
   property :id, Serial, key: true
   property :name, String
-  has n, :dishes, :model => 'Dish', :child_key => [:categoryId]
+  has n, :dishes
```

Comment:
Tried to understand how data binding works than having a relation to a different object. This is what we came up with. Didn't know it could be done so much more simple.